xrpl/asyncio/transaction/reliable_submission.py

```diff
@@ -89,6 +89,8 @@ async def send_reliable_submission(
     transaction_hash = transaction.get_hash()
     submit_response = await submit_transaction(transaction, client)
     prelim_result = submit_response.result["engine_result"]
+    if prelim_result != 'tesSUCCESS':
```

Comment:
This should only check for `temMALFORMED` or any of the error codes that don't propagate across the peer network. There are some "error" responses that eventually "go away" and the transaction becomes valid - such as `telINSUF_FEE_P`.